setup.cfg

```diff
@@ -94,6 +92,8 @@ ignore =
     D415
     # E231 missing whitespace after ','
     E231
+    # E704 multiple statements on one line (def)
+    E704
```

Comment:
These Black related linting changes most probably has nothing to do with dropping Python 3.7 support. @hugovk probably this was added due to running a recent Black version on the Darker codebase, or is there some other reason?

I'd prefer to collect formatting updates in a separate PR.

Reply:
Yes, from Black.